contrib/ansible/roles/node/tasks/main.yml

```diff
@@ -1,13 +1,7 @@
 ---
-- name: Check if selinux enforcing
-  command: getenforce
-  register: selinux
-  changed_when: false
-  when: ansible_selinux
-
 - name: Set selinux permissive because tokens and selinux don't work together
   selinux: state=permissive policy=targeted
```

Comment:
maybe, while you are at it, use ansible_selinux.type here, instead of hard coding to 'targeted' ?